src/lib/formats/flopimg.cpp

```diff
@@ -690,17 +654,14 @@ floperr_t floppy_format_track(floppy_image_legacy *floppy, int head, int track,
 	/* create a dummy resolution; if no parameters were specified */
 	if (!parameters)
 	{
-		alloc_resolution = std::make_unique<util::option_resolution>(floppy_option_guide, floppy->floppy_option->param_guidelines);
+		alloc_resolution = std::make_unique<util::option_resolution>(floppy_option_guide);
```

Comment:
Oh another one of these - you need to change it to a `try { } catch (...) { }` because make_unique never returns a null pointer.